models/src/main/scala/coop/rchain/models/BitSetBytesMapper.scala

```diff
@@ -0,0 +1,40 @@
+package coop.rchain.models
+
+import com.trueaccord.scalapb.TypeMapper
+
+import scala.collection.immutable.BitSet
+import java.nio.ByteBuffer
+
+import com.google.protobuf.ByteString
+
+object BitSetBytesMapper {
+  val BYTES_PER_LONG = 8
+
+  implicit val bitSetBytesMapper: TypeMapper[ByteString, BitSet] =
+    TypeMapper(
+      byteStringToBitSet
+    )(bitSetToByteString)
+
+  def bitSetToByteString(bitset: BitSet): ByteString = {
+    val longs: Array[Long] = bitset.toBitMask
+    val bytes: Array[Byte] = (ByteBuffer.allocate(BYTES_PER_LONG * longs.length) /: longs) {
```

Comment:
I'd avoid using the infix symbolic version of `foldLeft`.  It demands that the reader is familiar with this quirky syntactic sugar or willing to go doc-diving, whereas it's a lot easier for most people to grok `foldLeft` upon initial read.

Reply:
Hmm we've used this throughout the normalizer/sorting code. It is too bad because I really like this sugar.

Reply:
That is too bad.

Reply:
Done.